### ISSUE TITLE
Reduce the CPU usage when nothing is going on

### ISSUE THREAD

**Issue by ntadej** [HSF/phoenix#205]
It seems Phoenix keeps doing high-FPS rendering even if nothing is going on (no animation, no user action). It might be useful to prevent this and only re-render when needed.

**Comment by EdwardMoyse** [comment]
Actually this is a good point - thanks @ntadej. Let me ping @9inpachi since I know he has been modifying the animation loop recently and might have an idea of how feasible this is to do...

**Comment by 9inpachi** [comment]
Indeed. I also thought of implementing this after seeing a game that only renders when controls are used.
We could have a similar render function only on either camera movement or when the animation runs.
Let's keep this issue as a reminder of implementing this.
Thanks @ntadej

**Comment by 9inpachi** [comment]
See #233 for an attempt.

**Comment by EdwardMoyse** [comment]
This is interesting:
https://discoverthreejs.com/book/first-steps/animation-loop/

(see render on demand)

**Comment by 9inpachi** [comment]
> (see render on demand)

We already have this in our `ThreeManager`. See [this function](https://github.com/HSF/phoenix/blob/01d5d86c3d6f91505b346083b070161bdcb348a3/packages/phoenix-event-display/src/three/index.ts#L168). The hard part is to actually call it in tons of places and then manage every update.
